charts/rstudio-library/templates/_rbac.tpl

```diff
@@ -12,14 +12,14 @@
 */ -}}
 {{- define "rstudio-library.rbac" -}}
 {{- $serviceAccountAnnotations := default (dict) .serviceAccountAnnotations }}
-{{- $serviceAccountCreate := default true .serviceAccountCreate }}
+{{- $serviceAccountCreate := or ( and (not (kindIs "bool" .serviceAccountCreate)) (empty .serviceAccountCreate) ) (not (not .serviceAccountCreate)) }}
```

Comment:
It like to take another look/whack at this. If we can do some boolean algebra to invert portions of this logic I think it could me more understandable.

My logic missed the case where it was unset and we wanted to create the service account.

Reply:
After our conversation I'd prefer that we implement this as a type check as in https://github.com/rstudio/helm/blob/main/charts/rstudio-library/templates/_debug.tpl

Or we could move all values validation to a JSON schema: https://github.com/rstudio/helm/issues/35